packages/react-admin-color-picker/src/core/ColorPicker.tsx

```diff
@@ -0,0 +1,89 @@
+import * as tinycolor from "tinycolor2";
+import { colorToHex } from "../utils/colorSpaces";
+import styles from "./ColorPicker.styles";
+import HexInput from "./HexInput";
+import Palette from "./Palette";
+import Picker from "./Picker";
+
+export interface IVPAdminColorPickerProps {
+    classes: {
+        input: string;
+        popper: string;
+        pickedColorIndicator: string;
+        saturationWrapper: string;
+        saturationPointer: string;
+        hueWrapper: string;
+        hueSliderMarker: string;
+        paletteWrapper: string;
+        paletteItem: string;
+        readOnlyInput: string;
+    };
+}
+
+interface IComponentProps extends FieldRenderProps<string, HTMLInputElement> {
+    colorPalette?: string[];
+    showPicker?: boolean;
+    pickerWidth?: number;
+}
+
+const ColorPicker: React.FC<IComponentProps & IVPAdminColorPickerProps> = ({
+    colorPalette,
+    showPicker,
+    pickerWidth,
+    classes,
+    input: { value, onChange },
+}) => {
+    const [anchorEl, setAnchorEl] = React.useState<HTMLInputElement | null>(null);
+    const [inputWidth, setInputWidth] = React.useState<number>(pickerWidth ? pickerWidth : 300);
+    const inputRef = React.useRef<HTMLInputElement>();
+
+    const handleAwayClick = () => {
+        setAnchorEl(null);
+    };
+
+    const handleFieldClick = (event: React.MouseEvent) => {
+        setAnchorEl(event.currentTarget as HTMLInputElement);
+    };
+
+    React.useEffect(() => {
+        onChange(colorToHex(value)); // convert initial color to hex
+    }, []);
+
+    React.useEffect(() => {
+        if (!pickerWidth && inputRef.current) setInputWidth(inputRef.current.offsetWidth);
+    }, [inputRef]);
+
+    const isOpen = Boolean(anchorEl);
+
+    return (
+        <ClickAwayListener onClickAway={handleAwayClick}>
+            <div>
+                <InputBase
+                    ref={inputRef}
+                    inputComponent={HexInput as React.ComponentType}
+                    value={value ? tinycolor(value).toHexString() : ""}
+                    inputProps={{
+                        value: value ? tinycolor(value).toHexString() : "",
+                        classes,
+                        picker: !!showPicker,
+                        palette: !!colorPalette?.length,
+                        pickerWidth,
+                    }}
+                    onChange={newColor => onChange(colorToHex((newColor as unknown) as tinycolor.ColorInputWithoutInstance))}
+                    className={classes.input}
+                    onClick={handleFieldClick}
+                />
+                <Popper open={isOpen} anchorEl={anchorEl} placement={"bottom"} style={{ width: `${inputWidth}px` }} className={classes.popper}>
+                    {showPicker && <Picker classes={classes} color={value} onChange={onChange} />}
+                    {colorPalette?.length && <Palette classes={classes} colors={colorPalette} onChange={onChange} />}
```

Comment:
Du hast jetzt im `<Palette />` einen Paper-Wrapper, der sollte glaub ich stattdessen um den gesamten Popper-Content gehen, sonst ist ja nur der untere Teil vom Popup gewrappt.

```suggestion
                    <Paper className={classes.popperContent}>
                        {showPicker && <Picker classes={classes} color={value} onChange={onChange} />}
                        {colorPalette?.length && <Palette classes={classes} colors={colorPalette} onChange={onChange} />}
                    </Paper>
```